CharlieBackend.Core/Entities/Homework.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CharlieBackend.Core.Entities
+{
+    public partial class Homework : BaseEntity
+    {
+        public ushort? DeadlineDays { get; set; }
```

Comment:
If it's deadlineDays, we need to count from some starting point. Also, we can't set particular time of deadline.
Let's use DateTime? DueDate